docs/source/install.rst

```diff
@@ -12,15 +12,19 @@ Conda
 
 Dask is installed by default in `Anaconda <https://www.anaconda.com/download/>`_::
 
-You can update Dask using `conda <https://www.anaconda.com/download/>`_::
+You can update Dask using the `conda <https://www.anaconda.com/download/>`_: command:
 
-    conda install dask
+   ``conda install dask``
```

Comment:
Why this change?  Why not stick to standard code block with the double colon?

Reply:
Ok, change reverted. Sorry, a bit new to the rst format, more used to md markdown (probably not a good excuse though 😝 ).